pycondor/dagman.py

```diff
@@ -404,12 +468,15 @@ def build_submit(self, makedirs=True, fancyname=True, submit_options=None):
             <http://research.cs.wisc.edu/htcondor/manual/current/condor_submit_dag.html>`_
             for possible options).
 
+        single_file: bool, optional
+            When it's true, single job submission file will be generated.  Default is False.
```

Comment:
Minor comment. Can you update the `single_file` description to describe what happens when `single_file=True` and when `single_file=False`. Perhaps something like:

```suggestion
            Option to create a single ``Job`` submit file. If ``True``, a single ``Job`` submit file will be created and variables for each ``Job`` will be defined in the corresponding ``Dagman`` submit file. If ``False``, then a submit file is generated for each ``Job`` being managed by a ``Dagman`` (default is ``False``).
```